dask/dataframe/tests/test_dataframe.py

```diff
@@ -50,6 +51,18 @@
 )
 d = dd.DataFrame(dsk, "x", meta, [0, 5, 9, 9])
 full = d.compute()
+
+dsk2 = {
+    ("x", 0): pd.DataFrame({"a": [[1, 2, 3]], "b": [4, 5, 6]}, index=[0, 1, 3]),
+    ("x", 1): pd.DataFrame({"a": [4, 5, 6], "b": [[3, 4, 5]]}, index=[5, 6, 8]),
+    ("x", 2): pd.DataFrame({"a": [7, 8, 9], "b": [0, 0, 0]}, index=[9, 9, 9]),
+}
+meta2 = make_meta(
+    {"a": "i8", "b": "i8"}, index=pd.Index([], "i8"), parent_meta=pd.DataFrame()
+)
+d2 = dd.DataFrame(dsk2, "x", meta2, [0, 5, 9, 9])
+full2 = d2.compute()
```

Comment:
This looks identical to the above dataframe `d`, and is unused below. Is there a reason to make another?

A meta point -- the above snippet constructing a dataframe looks quite old, and for new tests I would probably advocate for using `dd.from_pandas()` rather than doing manual construction.

Reply:
Done.